packages/server-wallet/deployment/deploy.ts

```diff
@@ -1,9 +1,10 @@
 import {Address} from '@statechannels/client-api-schema';
 import {ContractArtifacts} from '@statechannels/nitro-protocol';
 import {GanacheDeployer} from '@statechannels/devtools';
-import {defaultConfig} from '../src/config';
 import {Wallet} from 'ethers';
 
+import {defaultTestConfig} from '../src/config';
```

Comment:
Is `deploy.ts` a test-specific deploy file? If yes, should it be renamed `deploy.test.ts`?